content/api/game-state.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Game State
+---
+
+## Mode
+Your robot can be in 1 of 2 modes: `DEVELOPMENT` and `COMPETITION`. By default, your robot will be in `DEVELOPMENT` mode. To work out which mode you're in, you need to compare the values against the `GameMode`:
+
+```python
+from robot.game import GameMode
```

Comment:
Is this not re-exported by the `robot` module?
I think we should encourage top-level module use only.

Reply:
ah, yes, you are correct.